ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -672,7 +672,7 @@ export const AdminInstitutionEdit = fp.flow(withNavigation, withRouter)(class ex
                 onBlur={v => this.setState(fp.set(['institution', 'displayName'], v.trim()))}
             />
             <div style={{color: colors.danger}} data-test-id='displayNameError'>
-              {!this.isAddInstitutionMode && errors && errors.displayName}
```

Comment:
this was the minor bug: did not complain when the displayName was too long in Add Mode.

Reply:
The Save button was correctly disabled.